voici/app.py

```diff
@@ -25,6 +24,7 @@
     strip_sources="VoilaConfiguration.strip_sources",
     template="VoilaConfiguration.template",
     theme="VoilaConfiguration.theme",
+    **{"classic-tree": "VoilaConfiguration.classic_tree"},
```

Comment:
```suggestion
    "classic-tree"="VoilaConfiguration.classic_tree",
```

Reply:
I need to use this trick because of the hyphen in the parameter name